GoogleSignIn/Tests/Unit/GIDAppCheckTest.m

```diff
@@ -57,9 +57,9 @@ - (void)testGetLimitedUseTokenFailure {
   GIDAppCheck *appCheck = [[GIDAppCheck alloc] initWithAppCheckProvider:fakeProvider
                                                            userDefaults:self.userDefaults];
 
-  [appCheck getLimitedUseTokenWithCompletion:^(id<GACAppCheckTokenProtocol> _Nullable token,
+  [appCheck getLimitedUseTokenWithCompletion:^(GACAppCheckToken * _Nullable token,
                                                NSError * _Nullable error) {
-    XCTAssertNil(token);
+    XCTAssertNotNil(token); // If there is an error, we expect a placeholder token
```

Comment:
nit: How about also checking if the token is the placeholder token?

Reply:
I don't know what comparison that would be since the token will look and feel like a regular token, and it will be created and provided by AppCheckCore.